datafusion/src/error.rs

```diff
@@ -74,6 +74,8 @@ pub enum DataFusionError {
 
 impl DataFusionError {
     /// Wraps this [DataFusionError] as an [arrow::error::ArrowError].
+    ///
+    /// TODO this can be removed in favor if the conversion below
```

Comment:
see https://github.com/apache/arrow-datafusion/pull/1645

Reply:
```suggestion
    /// TODO this can be removed in favor of the conversion below
```

Why not remove this now? If we want to keep it, then it would be better to add a `deprecated` attribute.

Reply:
Sorry -- I removed it in https://github.com/apache/arrow-datafusion/pull/1645 (probably wasn't worth leaving a TODO) 🤔